prime-router/metadata/HL7/catchall/hl7/segments/PRT/Observation-Location.yml

```diff
@@ -0,0 +1,60 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+# TODO need message level conditions to be able to implement FHIR > HL7 mapping
+
+# This segment does not appear in NIST or RADx MARS
+
+resourceType: Observation
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+extension:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - expressionType: nested
+      specs: PRT.9 *
+      expressionsMap:
+        url:
+          type: STRING
+          value: "http://hl7.org/fhir/StructureDefinition/event-location"
+        valueReference:
+          expressionType: reference
+          valueOf: datatypes/PL/Location
+        extension:
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: hl7v2Field
+            valueString:
+              type: STRING
+              valueOf: PRT.9
+    - expressionType: nested
+      specs: PRT.14 *
+      expressionsMap:
+        url:
+          type: STRING
+          value: "http://hl7.org/fhir/StructureDefinition/event-location"
+        valueReference:
+          expressionType: reference
+          valueOf: datatypes/XAD/Address
+        extension:
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: hl7v2Field
+            valueString:
+              type: STRING
+              valueOf: PRT.14
+    - expressionType: resource
+      valueOf: segments/PRT/PRTExtension
+
+subject:
```

Comment:
I'm kinda confused by how this is presented in the mapping inventory. If it's meant to be an Observation instead of a location it's missing some required fields in the FHIR spec (Observation.code and status currently). The closest example I could find was [PD1[Observation-LivingWill]](https://docs.google.com/spreadsheets/d/1v2Ip_zAEz8INIx7tLmTa4PwyG6IfEUD1wjzVLJls4Ik/edit?gid=0#gid=0) which specifies a hardcoded status and code. But I'm also commenting here as I'm curious why we're mapping subject here. (And maybe these are just things that will be go over when these mappings are enabled and updated).

Reply:
I think we both have the same question; thinking on it some more, it does make more sense that this creates a Location resource that the parent OBX references. I was planning on deferring this to the followup issue instead of holding PRT[Device] up.

Reply:
Yeah let's kick this to the next ticket. I don't think we have a hard or official stance on generating FHIR that is invalid but I feel like there must be something we can do in this case.